src/Components/InfoBox.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+
+class Display extends React.Component {
```

Comment:
Because of what this component is doing and returning, you can change this from a class based component into a function based component.

Here's an article that explains the difference between the two and when it would be more appropriate to use each https://medium.com/@Zwenza/functional-vs-class-components-in-react-231e3fbd7108

Reply:
Migrated to functions where appropriate